app/controllers/admin/users_controller.rb

```diff
@@ -91,20 +101,47 @@ def create
             m.course = @course
           }
           raise ActiveRecord::Rollback unless membership.save
+          
+          # if user is guest, create a entry under guest database as well
+          if params[:user]["course_role"] == '3'
```

Comment:
Use SEMANTIC_CONSTANT instead of naked char entry.  Shouldn't this be an integer (3) rather than char ('3')?

Reply:
Have changed it to use a SEMANTIC_CONSTANT.